src/cli.ts

```diff
@@ -300,20 +300,54 @@ See more help with --help`,
     if (!Flags.skipValidation) {
       const validatorSpinner = ora("Validating generated types").start();
       if (Flags.all) validatorSpinner.indent = 1;
+
+      const extraFiles = [];
+      for (const io of inputOutputMappings) {
+        if (getImportPath(inputPath, io.input) !== "/") {
+          try {
+            const fileInputPath = join(process.cwd(), io.input);
+            const inputFile = await readFile(fileInputPath, "utf-8");
+            extraFiles.push({
+              sourceText: inputFile,
+              relativePath: io.input,
+            });
+          } catch {
+            validatorSpinner.warn(`File "${io.input}" not found`);
+          }
+
+          try {
+            const fileOutputPath = join(process.cwd(), io.output);
+            const outputFile = await readFile(fileOutputPath, "utf-8");
+            extraFiles.push({
+              sourceText: outputFile,
+              relativePath: io.output,
+            });
+          } catch {
+            validatorSpinner.warn(
+              `File "${io.output}" not found: maybe it hasn't been generated yet?`
```

Comment:
when re-running https://github.com/fabien0102/ts-to-zod/commit/502b7c08233c33c60f09c298d76d0a756a170b7b on top of your PR, I noticed that the first time I ran

```shell
yarn gen:all
```

I got this warning:

```
Generating "example"
Generating "example/person"
Generating "config"
Generating "description"
Generating "bug"
 ⚠ File "example/bug.zod.ts" not found: maybe it hasn't been generated yet?
 ⚠ File "example/bug.zod.ts" not found: maybe it hasn't been generated yet?
 ⚠ File "example/bug.zod.ts" not found: maybe it hasn't been generated yet?
 ⚠ File "example/bug.zod.ts" not found: maybe it hasn't been generated yet?
``` 

So this now tries to load all output files, regardless whether they are referenced by a configuration?
Ideally:
1. we would have a dependency tree if multiple configurations are generated, so that configurations that depend on others await their generation
2. only issue relevant warnings

Reply:
Yes, this is only for first full generation (for projects which rely on that). 
Building the whole dependency tree and ordering generations based on it is a whole feature and not part of this big IMO.

Reply:
definitely a separate feature, I did not expect this to be part of this fix